tests/profiling_v2/collector/test_memalloc.py

```diff
@@ -58,7 +58,7 @@ def test_memory_collector(tmp_path):
     )
     ddup.start()
 
-    mc = memalloc.MemoryCollector(None)
+    mc = memalloc.MemoryCollector()
```

Comment:
The MemoryCollector `__init__` function takes the recorder as a positional argument so this won't work

Reply:
Updated `MemoryCollector`